README.md

```diff
@@ -621,31 +621,55 @@ the [ENCODE ATAC-seq data standards](https://www.encodeproject.org/atac-seq/#sta
+| TSS enrichment < 4                                                                                                 | align\_enrich.tss\_enrich.tss\_enrich                             | This cutoff needs to be evaluated retrospectively. We will probably have tissue-specific recommendations                                                             |
 
-## 6. Post-processing scripts
+## 6. Pipeline post-processing scripts
 
-Use the post-processing wrapper scripts to generate the QC report and to generate the final count matrix.
+Use the pipeline post-processing wrapper scripts to organize pipeline outputs, generate the QC report and  the final consensus peak by count matrix. 
 
-* [atac-post-process-wrapper.sh](src/atac-post-process-pass-wrapper.sh): wrapper script to generate QC report and final
-  count matrix for PASS samples
-* [atac-seq-human-wrapper.sh](src/atac-post-porcess-human-wrapper.sh): wrapper script to generate QC report and final
-  count matrix for human samples
+### 6.1. Pipeline post-processing wrappers
+ 
+Below wrappers can be used to generate the consensus peak*counts matrix for all tissues combined together
+
+* [atac-post-process-wrapper.sh](src/atac-post-process-pass-wrapper.sh): wrapper script to generate QC report and final count matrix for PASS animal studies
+* [atac-seq-human-wrapper.sh](src/atac-post-porcess-human-wrapper.sh): wrapper script to generate QC report and final count matrix for human study
 
 For each of these wrappers, make sure you fill in the appropriate variables at the top of the script before running.
 
-### 6.1. Individual scripts
+#### Individual scripts contained in the wrapper
 
 * [extract\_rep\_names\_from\_encode.sh](src/extract_rep_names_from_encode.sh): generate rep-to-viallabel map to
   interpret QC report
-* [pass\_extract\_atac\_from\_gcp.sh](src/pass_extract_atac_from_gcp.sh): download relevant files for PASS samples from
-  ENCODE output
-* [human\_extract\_atac\_from\_gcp.sh](src/extract_atac_from_gcp_human.sh): download relevant files for human samples
-  from ENCODE output
+* [pass\_extract\_atac\_from\_gcp.sh](src/pass_extract_atac_from_gcp.sh): download relevant files for PASS samples from ENCODE pipeline outputs
+* [human\_extract\_atac\_from\_gcp.sh](src/extract_atac_from_gcp_human.sh): download relevant files for human samples from ENCODE pipeline outputs
 * [encode\_to\_count\_matrix.sh](src/encode_to_count_matrix.sh): use `narrowkpeak.gz` and `tagAlign` files to generate a
   peak x sample raw counts matrix for PASS samples
 * [encode\_to\_count\_matrix_human.sh](src/encode_to_count_matrix_human.sh): use `narrowkpeak.gz` and `tagAlign` files
   to generate a peak x sample raw counts matrix for human samples
 * [align\_stats.sh](src/align_stats.sh): calculate % of primary alignments aligning to chrX, chrY, chrM, autosomes, and
   contigs
 * [merge\_atac\_qc.R](src/merge_atac_qc.R): merge wet lab QC, curated pipeline QC, and alignment stats
+
+### 6.2 Custom analysis
```

Comment:
I think 6.1 is about processing all tissues simultaneously and 6.2 is for each tissue individually. Something like:

6.1. Generate consensus merged peak and count matrix for all tissues combined
6.2. Generate tissue-specific consensus merged peak and count matrix. 

And please, adjust the text to explain it in that way